### DISCUSSION TITLE
Adjust usage of address refresh icon & consider automatically updating the address

### DISCUSSION THREAD

**Discussion by sdellava** [iotaledger/firefly#1545] [Features / Enhancements]
## Issue description

Due to the address can be changed only after it has been used, the icon that allow the user to renew it, is misleading. 

## Wallet version

0.3.2

- wallet version (version number, commit, or branch):

## System specification

Win

## Is it a repeatable issue?

Y

## Steps To reproduce the issue

Explain how the maintainer can reproduce the issue.

1. click on the address renewal icon (doble rounded arrow)
2. the icon rotate
3. id the address has never been used, nothing happen

## Expected behaviour

If the address has never been used, the icon should be disable.

**Comment by dev0pa** [comment]
If an address has already been used, a new one should appear automatically. A manual update is unnecessary.
